index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=, initial-scale=1.0" />
+    <title>Portfolio|Dahir-Mohamed</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="./assets/style.css" />
+  </head>
+
+  <body>
+    <!-- The logo and nav-links are nested in a header tag  -->
+
+    <header class="header">
+      <h1 class="logo">PORTFOLIO</h1>
+
+      <nav class="nav-bar-link">
+        <div class="nav-link"><a class="link" href="#about">About Me</a></div>
+        <div class="nav-link">
+          <a class="link" href="#projects">Projects</a>
+        </div>
+        <div class="nav-link">
+          <a class="link" href="#contact">Contact Me</a>
+        </div>
+      </nav>
+    </header>
+
+    <!-- The h1 tag containing my name is nested in a div tag that also contains a background img   -->
+
+    <div class="banner">
+      <h1 class="name">Dahir A Mohamed</h1>
+    </div>
+
+    <!-- The main content of the webpage which is the about me, projects and contact me section is all nested in a main tag -->
+
+    <main>
+      <!--  the about me section contains an avatar, about me title and contents which is all nested in a section tag -->
+
+      <section class="about-me-img-container">
+        <div class="img-section">
+          <img
+            src="https://cdn5.vectorstock.com/i/1000x1000/63/09/job-icon-male-person-worker-avatar-symbol-desk-vector-23636309.jpg"
+            alt="self-portrait"
```

Comment:
This image isn't showing on your site. Consider uploading it elsewhere or adding as an image in your assets folder.